target/linux/lantiq/base-files/lib/preinit/06_init_network_lantiq

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+. /lib/functions/lantiq.sh
+
+set_preinit_misc_lantiq() {
+board=$(board_name)
```

Comment:
Indent!

I wonder why after so many devices in lantiq this would be the first one requires such an additional setup.